sharespots/settings/base.py

```diff
@@ -6,8 +6,15 @@
     get_env,
 )
 
+from dotenv import load_dotenv
+
```

Comment:
load_dotenv() -- working fine when adding this line as per docs / showing error w\ it

Reply:
What error are you getting?

Reply:
this is the err without load_dotenv() django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.

Reply:
I have the followoing  lines that run load_env:
if os.path.exists(ENV_FILE):
    load_dotenv(dotenv_path=ENV_FILE,override=True)

Can you debug and check if your code is making it past the if statement. 

This setup works for me on windows but haven't confirmed on my mac

Reply:
Everything works fine now!